include/config.h

```diff
@@ -295,6 +295,15 @@
 
 #define RESEED_RNG 100000
 
+/* The amount of entries in the testcase cache, held in memory.
+Decrease if RAM usage is high. */
+#define TESTCASE_CACHE_SIZE 2048
```

Comment:
if we add this, this should be an env (and setting the env to any size should enable the feature). or?